memgpt/llm_api_tools.py

```diff
@@ -40,21 +40,21 @@ def is_context_overflow_error(exception):
             try:
                 error_details = exception.response.json()
                 if "error" not in error_details:
-                    printd(f"HTTPError occured, but couldn't find error field: {error_details}")
+                    printd(f"HTTPError ocurred, but couldn't find error field: {error_details}")

```

Comment:
both here are mispelled, should be "occurred" afaik

Reply:
Thanks! Changes have been made.